### ISSUE TITLE
ScalaFX as a Typelevel Project

### ISSUE THREAD

**Issue by rajmahendra** [scalafx/scalafx#250]
I had a talk with @jpsacha  and planned to initiate the steps to make ScalaFX as part of Typelevel project. https://github.com/typelevel/general/issues/46

Typelevel wants following things from our contributors/Committers 

> (1) There must be a consensus amongst the project's stakeholders that they want to join Typelevel.

Please post who are all like to join Typelevel as part of ScalaFx project

> (2) The project must be willing to support the Typelevel code of conduct or something equivalent.

http://typelevel.org/about.html

**Comment by jpsacha** [comment]
I still not completely clear what it means to "join" typelevel. Is this just listing ScalaFX on their website. Is there some more involvement beyond that?

BTW: I do not have a problem with the code of conduct. ScalaFX forums are very civil.

**Comment by rajmahendra** [comment]
@jpsacha It is not we move all into Typelevel. its all about adopting the CoC and Goal of the Tl

**Comment by rajmahendra** [comment]
@Alain-Bearez @andy1138 @beolnix @rafonso @svenreimers @steveonjava  Your view on this issues. 
@steveonjava and @jpsacha  are ok for this.

**Comment by Alain-Bearez** [comment]
I agree to join Typelevel.

# TL;DR

Having the project under the Typelevel umbrella would give it more visibility, and maybe attract more contributors. I feel sorry I did not contribute a single line of code during the last three years or so.

On the other hand, the Typelevel projects are usually "more abstract", as they are offering more of "hard-core" functional programming libraries.

I would say it depends more on *them* welcoming our project, rather than the other way around.

**Comment by rajmahendra** [comment]
@Alain-Bearez  thank you. I have posted in the Typelevel issue.

**Comment by jpsacha** [comment]
I think we can close this issue since TypeLevel seem not to like JavaFX as being imperative (https://github.com/typelevel/general/issues/46#issuecomment-282260485).
